fdbclient/NativeAPI.actor.cpp

```diff
@@ -3302,9 +3340,13 @@ Future< std::pair<Optional<StorageMetrics>, int> > Transaction::waitStorageMetri
 }
 
 Future< StorageMetrics > Transaction::getStorageMetrics( KeyRange const& keys, int shardLimit ) {
-	StorageMetrics m;
-	m.bytes = -1;
-	return extractMetrics( ::waitStorageMetrics( cx, keys, StorageMetrics(), m, StorageMetrics(), shardLimit, -1 ) );
+	if (shardLimit > 0) {
```

Comment:
I assume a non-positive shard limit means unlimited? Could you add a comment to this function indicating that?

Reply:
Yes. Sure.